src/idiom340ex.fz

```diff
@@ -0,0 +1,14 @@
+#fuzion_idiom340
+
+#Define the string 's' as "example" 
+#variable 'c' whose value debends on 's'
+s := "example"
+c :=
+
+  if s.is_empty then       #Check if the string is empty
+    "Empty string"         #If it's empty, set the value to "Empty string"
+  else
+    s.substring_codepoint(s.codepoint_length - 1, s.codepoint_length)   #If it's not empty, get the last character using codepoint indexing
```

Comment:
It is preferred to not use `(`/`)` in calls in Fuzion. 
Also, there is a version of `substring_codepoint` with only the `from` argument that would suffice here. So my suggestion:
```suggestion
    s.substring_codepoint s.codepoint_length-1   # If it's not empty, get the last character using codepoint indexing
```
or, if you prefer spaces around `-` 
```suggestion
    s.substring_codepoint (s.codepoint_length - 1)   # If it's not empty, get the last character using codepoint indexing
```
(please note the space before `(`).